packages/shared/src/components/cards/ActionButtons.tsx

```diff
@@ -163,7 +163,11 @@ export default function ActionButtons({
         >
           <ReadArticleButton
             className="mr-2 btn-primary"
-            href={post.permalink}
+            href={
+              post.type === PostType.Share
+                ? post.sharedPost.permalink
+                : post.permalink
+            }
```

Comment:
I think we also need to change the openNewTab if we look at the `SharePostContent` handler for instance:
```
href={
                isSharedPostSquadPost
                  ? post.sharedPost.commentsPermalink
                  : post.sharedPost.permalink
              }
              openNewTab={isSharedPostSquadPost ? false : openNewTab}
```

Reply:
That's when for when the shared post is to a squad post. Don't think this is an issue here, haven't encountered it. If I do, I'll raise a PR to fix it.